src/main/java/frc/robot/subsystems/CDSSubsystem.java

```diff
@@ -235,24 +246,47 @@ public int getBallCount() {
     return ballCount;
   }
 
-  public boolean getReady() {
-    return isReady;
-  }
-
-  public void setReady(boolean status) {
-    isReady = status;
+  public ManagementState getState() {
+    return state;
   }
 
-  public void periodic() {}
-
-  private void instantiateDebugTab() {
-    CDSTab = Shuffleboard.getTab("CDS Tab");
-    CDSBeltControllerDirection =
-        CDSTab.add("CDS Belt Direction", "Not Running")
-            .withPosition(2, 0)
-            .withWidget(BuiltInWidgets.kToggleSwitch)
-            .getEntry();
-    CDSWheelControllerSpeed = CDSTab.add("CDS Wheel speed", 0).withPosition(3, 0).getEntry();
-    CDSBeltControllerSpeed = CDSTab.add("CDS Belt speed", 0).withPosition(4, 0).getEntry();
+  public void periodic() {
+    int ballCount = getBallCount();
```

Comment:
You should be able to just use the instance variable ballCount instead of redefining it here